### ISSUE TITLE
any python version？

### ISSUE THREAD

**Issue by jhon353** [jMotif/SAX#28]
any python version？

**Comment by seninp** [comment]
Yes — there's a Python implementation: https://github.com/seninp/saxpy (on PyPI as `saxpy`). It covers SAX, PAA, z-normalization, sliding-window SAX and HOT-SAX discord discovery.

This repository (jMotif/SAX) is the Java implementation. Closing as answered.
